pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -134,7 +134,7 @@ func createContainer(args []string) error {
 }
 
 func clusterIsIPv6(cfg *config.Cluster) bool {
-	return cfg.Networking.IPFamily == "ipv6"
```

Comment:
Dan was right, this is setting the API to listen in ::1 in a dual stack cluster, I have to fix it

Reply:
and podman?

Reply:
?? in which sense?

Reply:
ahh, why I didn't change in podman yo mean

Reply:
checked again, is ::1 in HA, I don't think that should be a problem right now, we can iterate later